### ISSUE TITLE
Might be a good idea to append a publisher to the installer

### ISSUE THREAD

**Issue by TsukiZero** [trobonox/kanri#531]
Or otherwise work it so Windows and Firefox doesn't think it's suspicious

**Comment by trobonox** [comment]
Thanks for your suggestion. I am aware of this missing, but it requires me to sign the app with a certificate (unless I am getting something wrong here), and the certificate which makes an app appear "trusted" is quite expensive.
As long as this project creates no profits and is 100% free, I can't afford to buy these certificates.

**Comment by trobonox** [comment]
> It appears to be through here: https://www.microsoft.com/en-us/wdsi/filesubmission/
> 
> But... IDK, requires a bit of research.

Good idea, I'll take a look and will do some research if I have some spare time.
